### PR TITLE
chakrashim: clean up legacy debugger code

### LINKED ISSUE #225: Remove deprecated debugger code
Now that inspector has landed, we need to remove the deprecated (and no longer functional) debugger code.

### PR DESCRIPTION
**Pull request by kfarnung**
Since debugger support was removed upstream, this change removes the
corresponding chakrashim code.

Resolves #225 

##### Checklist

- [x] `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes
- [x] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-message-guidelines)

##### Affected core subsystem(s)

chakrashim

### CODE REVIEW

**Review by kunalspathak — APPROVED** [review]

File: deps/chakrashim/src/v8context.cc [hunk 1]
```diff
@@ -80,10 +79,7 @@ Local<Context> Context::New(Isolate* external_isolate,
...
       external_isolate, static_cast<Context *>(context));
 
-  bool debugEnabled = jsrt::Debugger::IsDebugEnabled();
-  bool inspectorEnabled = jsrt::Inspector::IsInspectorEnabled();
-
-  if ((debugEnabled || inspectorEnabled) &&
+  if (jsrt::Inspector::IsInspectorEnabled() &&
       isoShim->debugContext == nullptr) {
     // If JavaScript debugging APIs need to be exposed then make sure
     // debugContext is available and chakra_debug.js or chakra_inspector.js
```
Comment (agarwal-sandeep) [inline deps/chakrashim/src/v8context.cc]: >chakra_debug [](start = 37, length = 12)

nit: remove

Comment (kfarnung) [reply deps/chakrashim/src/v8context.cc]: Fixed.

**Comment by agarwal-sandeep** [comment]
:shipit:

**Comment by kfarnung** [comment]
CI: https://ci.nodejs.org/job/chakracore-test-pull-request/43/

**Pull request merged**